FirebaseInAppMessaging/Swift/Tests/Integration/FIAMSwiftUI/Shared/ModalInAppMessageView.swift

```diff
@@ -58,11 +58,12 @@ struct ModalInAppMessageView: View {
   }
 
   // Need a dismiss button for the case where there's an action button with an action URL. Otherwise
-  // user would be forced into a clickthrough.
+  // user would be forced into a clickthrough. If there's No action button at all we also need a
```

Comment:
nit: I think "No" should be lowercase.
```suggestion
  // user would be forced into a clickthrough. If there's no action button at all we also need a
```